src/js/edu-benefits/1990e/config/form.js

```diff
@@ -126,6 +139,29 @@ const formConfig = {
     schoolSelection: {
       title: 'School Selection',
       pages: {
+        schoolSelection: {
+          title: 'School selection',
+          path: 'school-selection',
+          uiSchema: {
+            educationProgram: educationProgram.uiSchema,
+            educationObjective: {
+              'ui:title': 'Education or career goal (for example, “Get a bachelor’s degree in criminal justice” or “Get an HVAC technician certificate” or “Become a police officer.”)',
+              'ui:widget': 'textarea'
+            },
+            nonVaAssistance: {
+              'ui:title': 'Are you getting, or do you expect to get any money (including, but not limited to, federal tuition assistance) from the Armed Forces or public health services for any part of your coursework or training?',
+              'ui:widget': 'yesNo'
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              educationProgram: educationProgram.schema(),
+              educationObjective,
+              nonVaAssistance
```

Comment:
`educationObjective` and `nonVaAssitance` are also used in 1995; Could we make the `educationProgram.schema()` take a list of fields to include like `toursOfDuty.schema()`? The second parameter could be the required fields (which I almost did for the `toursOfDuty`). We could then put the `uiSchema` for these in that definitions file and not repeat it between forms.